src/python/devcontainer-feature.json

```diff
@@ -25,6 +25,11 @@
       "default": true,
       "description": "Install common Python tools like pylint"
     },
+    "installToolsExtra": {
+      "type": "string",
+      "default": "",
```

Comment:
Can you add an empty `proposals` array here? The schema can be seen in Typescript here: https://github.com/devcontainers/cli/blob/main/src/spec-configuration/containerFeaturesConfiguration.ts#L52-L72

"Proposals" are meant to act as an array of suggestions, but also indicates that the user can pass whatever argument they would like.  The "enum" variant, on the other hand, represents a the only set of permitted arguments.  (This isn't enforced in the CLI, but may be someday).

Reply:
Added empty `proposals`. 

Should I move the example in the description of the option to the `proposals` attribute?

```
proposals: [
   "poetry",
   "isort poetry sphinx"
]
```